tests/compute/half-texture.slang.expected

```diff
@@ -0,0 +1,185 @@
+result code = 0
```

Comment:
The same problem arises here as for the output HLSL: any small changes to how we emit GLSL and/or SPIR-V could break this output, and in the case of the SPIR-V output even updating the version of glslang we embed could conceivably change this output.

This is where we have the "cross-compile" tests where there is a `foo.slang` file together with a `foo.slang.glsl` and/or `foo.slang.hlsl` for comparison. Those at least let us just compare that the output DXBC/DXIL/SPIRV is the same for an end-to-end `slangc` invocation as for a matching fxc/dxc/glslang invocation on suitable HLSL/GLSL.

Even if it just involves taking the output HLSL/GLSL from Slang and dumping it to the `foo.slang.hlsl` or `foo.slang.glsl` file, that provides a (small) amount of insurance against changes to how Slang outputs source code, and against changes in how we invoke the downstream compilers.

I'm fine with checking these test outputs in as-is for now, but when some later change causes these tests to fail I would rather change their approach than just blindly update the baselines.